### PR TITLE
Showing a toast on back button press

### PR DESCRIPTION
**Pull request by AbsolutelySaurabh**
## Description

Now showing a toast saying "Please press back again to exit" on backpressed.
## Related issues and discussion
#735 
 
 ## Screen-shots, if any
 
 ## Checklist
 
 Please make sure these boxes are checked before submitting your pull request - thanks!
 
 
 
 - [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.
 
 - [x] If you have multiple commits please combine them into one commit by squashing them.
 
 - [
screenshot_20180216-014621
 ] Read and understood the contribution guidelines .

### CODE REVIEW

**Comment by itchix** [comment]
@Karljoones +1 for the log file

**Comment by AbsolutelySaurabh** [comment]
Kindly refer to the new PR I've made #777 with more changes implemented, I'll close this one if you'll say.

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/BaseActivity.java [hunk 1]
```diff
@@ -36,4 +41,24 @@ public void setContentView(@LayoutRes int layoutResID) {
...
+        this.doubleBackToExitPressedOnce = true;
+        Toast.makeText(this, "Please press back again to exit", Toast.LENGTH_SHORT).show();
+
+        new Handler().postDelayed(new Runnable() {
+
+            @Override
+            public void run() {
+                doubleBackToExitPressedOnce=false;
+            }
+        }, 1000);
```
Comment (Shubham2320) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/BaseActivity.java]: @AbsolutelySaurabh I guess the timer of the postDelayed `run` method should be same as the duration of Toast Message and as the duration of LENGTH_SHORT is 2000ms so I'd suggest doing same in `run` method too. :)

**Comment by Karljoones** [comment]
Closing this due to new PR being made

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/BaseActivity.java (hunk 1 above)
Comment (AbsolutelySaurabh) [reply app/src/main/java/openfoodfacts/github/scrachx/openfood/views/BaseActivity.java]: Ya, I've changed this too in the new PR #777 .

**Pull request closed without merging**